packages/design-system/src/components/Tooltip/Tooltip.tsx

```diff
@@ -1,7 +1,12 @@
-import React, { ReactElement } from 'react';
-import { useTooltipState } from 'reakit';
+import React from 'react';
+import {
+	useTooltipState as useReakitTooltipState,
+	Tooltip as ReakitTooltip,
+	TooltipArrow as ReakitTooltipArrow,
+	TooltipReference as ReakitTooltipReference,
+} from 'reakit';
 
-import * as S from './Tooltip.style';
+import theme from './Tooltip.module.scss';
```

Comment:
```suggestion
import styles from './Tooltip.module.scss';
```
Just to avoid confusing between this and what themes actually are